plugins/chat/assets/javascripts/discourse/components/chat/modal/edit-channel-name.gjs

```diff
@@ -54,10 +54,7 @@ export default class ChatModalEditChannelName extends Component {
       this.channel.title = result.channel.title;
       this.channel.slug = result.channel.slug;
       await this.args.closeModal();
-      await this.router.replaceWith(
-        "chat.channel",
-        ...this.channel.routeModels
-      );
+      this.router.replaceWith("chat.channel", ...this.channel.routeModels);
```

Comment:
When using `await` here it would cause a TransitionAborted error in drawer when editing channel name. I don't think we need this anyway